specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2020-09-01/examples/PrivateLinkResourcesList.json

```diff
@@ -0,0 +1,25 @@
+{
+  "parameters": {
+    "api-version": "2020-09-01",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "name": "management",
+            "type": "Microsoft.ContainerService/managedClusters/privateLinkResources",
+            "groupId": "management",
+            "requiredMembers": [
+              "management"
+            ],
+            "privateLinkServiceID": "/subscriptions/subid2/resourceGroups/rg2/providers/Microsoft.Network/privateLinkServices/plsName"
```

Comment:
PLS ID Not required on Swagger

Reply:
It would be more clear to have it here, and AKS already have logic to make sure PLSID is only exposed to NRP calls. I would like to have it as a part of swagger, unless you don't want to have swagger for these APIs.